examples/Readme.mdown

```diff
@@ -2,10 +2,16 @@
 
 All examples are small applications that you can go to it's root directory and run `npm install` to have it all setup.
 
-You can then run:
+For Mendel 1.x, you can run:
 
+    $ npm run build
     $ npm run development
 
+For Mendel 2.x, you can run:
+
+    $ npm run buildv2
+    $ npm run v2
```

Comment:
@stephanwlee really? I suggested so many times to keep this just on the npm install stuff. having two ways feels kinda sloppy